src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -176,13 +176,14 @@ function useChatUtilsHook(): UseChatUtilsHook {
     navigation.goBack();
   };
   const handleTextSystem = (item): string => {
+    let message = item?.description || item?.message;
     if (
       item?.rawJson?.userIdFollower === profile?.myProfile?.user_id ||
       item?.rawJson?.message?.userIdFollower === profile?.myProfile?.user_id
     ) {
-      return item?.rawJson?.textOwnMessage || item?.rawJson?.message?.textOwnMessage;
+      message = item?.rawJson?.textOwnMessage || item?.rawJson?.message?.textOwnMessage;
     }
-    return item?.description || item?.message;
+    return message;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for assigning the `message` variable has been reversed in the new hunk. This change could potentially introduce bugs if `item?.description` or `item?.message` are expected to be default values when `userIdFollower` does not match `user_id`. If this is not the intended behavior, consider reverting to the original order of assignment.
```diff
-    let message = item?.description || item?.message;
-    if (
-      item?.rawJson?.userIdFollower === profile?.myProfile?.user_id ||
-      item?.rawJson?.message?.userIdFollower === profile?.myProfile?.user_id
-    ) {
-      message = item?.rawJson?.textOwnMessage || item?.rawJson?.message?.textOwnMessage;
-    }
+    if (
+      item?.rawJson?.userIdFollower === profile?.myProfile?.user_id ||
+      item?.rawJson?.message?.userIdFollower === profile?.myProfile?.user_id
+    ) {
+      return item?.rawJson?.textOwnMessage || item?.rawJson?.message?.textOwnMessage;
+    }
+    return item?.description || item?.message;
```
Please ensure that this change aligns with your application's requirements and expectations.


<!-- This is an auto-generated comment by OSS CodeRabbit -->